CHANGELOG.md

```diff
@@ -8,7 +8,22 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 
 This update brings the below changes over the previous release:
 
+### Breaking changes over preview release v5.0.0-preview2
+
+- Dropped classes from the `Microsoft.Data.SqlClient.Server` namespace and replaced them with supported types from the [Microsoft.SqlServer.Server](https://github.com/dotnet/SqlClient/tree/main/src/Microsoft.SqlServer.Server) package.[#1585](https://github.com/dotnet/SqlClient/pull/1585) The affected classes and enums are:
```

Comment:
Does it necessary mentioning the .NET Core here?

Reply:
Didn't we drop them for all targets?

Reply:
Yes, but the change over previous ones is only dropping them for netcore project. We forwarded those types in netfx before.